other/oembed-addon/bootstrap.js

```diff
@@ -63,10 +63,10 @@ let hook = {
     let links = iframeDoc.getElementsByTagName("a");
     // Don't detect links in quotations.
     [x.skip = true
-      for each ([, x] in Iterator(iframeDoc.querySelectorAll("blockquote a")))];
+      for ([, x] of Iterator(iframeDoc.querySelectorAll("blockquote a")))];
```

Comment:
Hey, I'm still seeing Iterator here and below, am I looking at the right version?